### ISSUE TITLE
Error when supplying non-object values to `initializationOptions`

### ISSUE THREAD

**Issue by jcs090218** [valentjn/languagetool-languageserver#65]
Hi, I am trying to create a Emacs plugin for this language server. I am currently getting this error to get the server up and running. 
```
Apr 04, 2021 2:06:22 AM org.bsplines.ltexls.server.LtexLanguageServer initialize
INFO: ltex-ls 10.0.0 - initializing...
Apr 04, 2021 2:06:22 AM org.bsplines.ltexls.server.LtexLanguageServer initialize
WARNING: clientSupportsWorkDoneProgress = true
Apr 04, 2021 2:06:22 AM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: java.lang.reflect.InvocationTargetException
java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.lambda$null$0(GenericEndpoint.java:67)
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.request(GenericEndpoint.java:120)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.handleRequest(RemoteEndpoint.java:261)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.consume(RemoteEndpoint.java:190)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.handleMessage(StreamMessageProducer.java:194)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.listen(StreamMessageProducer.java:94)
	at org.eclipse.lsp4j.jsonrpc.json.ConcurrentMessageProcessor.run(ConcurrentMessageProcessor.java:113)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:832)
Caused by: java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.lambda$null$0(GenericEndpoint.java:65)
	... 11 more
Caused by: java.lang.ClassCastException: class com.google.gson.JsonNull cannot be cast to class com.google.gson.JsonObject (com.google.gson.JsonNull and com.google.gson.JsonObject are in unnamed module of loader 'app')
	at org.bsplines.ltexls.server.LtexLanguageServer.initialize(LtexLanguageServer.java:77)
	... 16 more
```

**Comment by valentjn** [comment]
The problem was that LT<sub>E</sub>X LS expected the value of `initializationOptions` during the [`initialize` request](https://microsoft.github.io/language-server-protocol/specification#initialize) either to be missing or to be a JSON object, but it seems to be given `null` according to the stack trace (which is a valid value according to the LSP spec). As a workaround, use `{}` for `initializationOptions`.

This should be fixed in [11.0.0-alpha.2](https://github.com/valentjn/ltex-ls/releases/tag/11.0.0-alpha.2).

**Comment by jcs090218** [comment]
Thanks! It works! :)

Here is the Emacs client for LTEX LS, https://github.com/emacs-languagetool/lsp-ltex.

**Comment by jcs090218** [comment]
And another Emacs client implementation using `eglot`, https://github.com/emacs-languagetool/eglot-ltex.

Just thought this might worth to mention since some users prefer `eglot` over `lsp-mode`.

**Comment by valentjn** [comment]
Great! I've added a list of language clients for LT<sub>E</sub>X LS to the readme.

**Comment by valentjn** [comment]
Fix released in 11.0.0. Please update as pre-releases are removed soon after the regular release.

**Comment by jcs090218** [comment]
Awesome! Just updated to `11.0.0`. See https://github.com/emacs-languagetool/lsp-ltex/commit/61cd3b287f5f7761f7dc0b48b3ed7a1a4a844a2c. :D
